internal/utils/proto/google_proto.go

```diff
@@ -0,0 +1,128 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+// Copied from https://github.com/kumahq/kuma/tree/9ea78e31147a855ac54a7a2c92c724ee9a75de46/pkg/util/proto
+// to avoid importing the entire kuma codebase breaking our go.mod file
+
+package proto
+
+import (
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+type (
```

Comment:
can we add the right code attributions
similar to https://github.com/istio/istio/blob/c71999b6bb65355de5195b4d8ce345f7e47ddc11/pkg/proto/merge/merge.go#L49 ?